dev/src/templates/JavaSpring401/libraries/spring-boot/openapi2SpringBoot.mustache

```diff
@@ -0,0 +1,178 @@
+package {{basePackage}};
+
+import com.fasterxml.jackson.databind.Module;
+
+import java.util.List;
+
+import org.openapitools.jackson.nullable.JsonNullableModule;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ExitCodeGenerator;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+{{^reactive}}
+import org.springframework.format.FormatterRegistry;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.validation.MessageCodesResolver;
+import org.springframework.validation.Validator;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.HandlerMethodReturnValueHandler;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.config.annotation.AsyncSupportConfigurer;
+import org.springframework.web.servlet.config.annotation.ContentNegotiationConfigurer;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+    {{^useSpringfox}}
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+    {{/useSpringfox}}
+import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.ViewResolverRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+    {{^java8}}
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+    {{/java8}}
+{{/reactive}}
+{{#reactive}}
+import org.springframework.web.reactive.config.CorsRegistry;
+    {{^useSpringfox}}
+import org.springframework.web.reactive.config.ResourceHandlerRegistry;
+    {{/useSpringfox}}
+import org.springframework.web.reactive.config.WebFluxConfigurer;
+{{/reactive}}
+
+@SpringBootApplication
+@ComponentScan(basePackages = {"{{basePackage}}", "{{apiPackage}}" , "{{configPackage}}"})
+public class OpenAPI2SpringBoot implements CommandLineRunner {
+
+    @Override
+    public void run(String... arg0) throws Exception {
+        if (arg0.length > 0 && arg0[0].equals("exitcode")) {
+            throw new ExitException();
+        }
+    }
+
+{{#mavenPomExists}}
+//    Uncomment and explicitly configure this main class.  For example, provide the <configuration><mainClass> or <start-class> elements in the pom.xml
```

Comment:
- Remove extra space between sentences.
- Add period after end of last sentence. If you're concerned about confusion resulting from `pom.xml.`, I'd say `pom.xml file.` instead. :)

Reply:
Maybe mention why users would want to uncomment and explicitly configure? For example, "For the OpenAPI endpoints to appear, uncomment and explicitly configure..."

Reply:
@sishida , one more thing about those commented out lines.  If, for example, you are outside of Codewind and you generate the spring server stub into an "empty" folder (at least with no pom.xml), then that main method will _NOT_ be commented out.  It is only commented out when we detect that a pom.xml file exists (it is a Maven project)

Reply:
Sounds like it would be good to tell users these instructions apply to Maven specifically. Maybe we could say something like, "For the OpenAPI endpoints to appear in a Maven project, uncomment..." I think this will suffice--because for these instructions to work, we need a pom.xml, and all Maven projects have pom.xml files...right?

Reply:
I think it is ok to _not_ mention that it is Maven, because it is already implied by the current context.